meta-dts-distro/recipes-bsp/packagegroups/packagegroup-txe-secure-boot.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "txe-secure-boot packagegroup"
+DESCRIPTION = "txe-secure-boot packagegroup"
+
+LICENSE = "MIT"
+
+inherit packagegroup
+
+PACKAGES = " \
+    packagegroup-txe-secure-boot \
```

Comment:
I am not sure if we need packagegroup here. How many more tools do we expect here? I think just one more, and it will not be public.

Reply:
I added it to ` ${PN}-tools-base`
https://github.com/Dasharo/meta-dts/commit/8f766cdf68746821e621fe2abffeee4df10c3e09#diff-b84bac204935ca0a7635a14b31645ba7e78426e334857f91d354f9b4013e1813R40